src/norm_layer_cuda.cu

```diff
@@ -749,7 +749,7 @@ __global__ void batchnorm2d_sample_var_post_processing(float const *data_in,
 {
     int col = blockIdx.x * blockDim.x + threadIdx.x;
     if (col < fi) {
-        data_out[col] = (data_in[col] + bias[col]) / scale;
+        data_out[col] = bias[col] / scale;
```

Comment:
These two are the correction you made in order to fix an error on the CNN+BatchNorm I introduced in a previous PR.